modules/ROOT/pages/persistence-gateway.adoc

```diff
@@ -0,0 +1,154 @@
+kind: Secret
+type: Opaque
+----
+
+Below are descriptions of the fields of the template you should modify:
+
+[%header,cols="2*a"]
+|===
+| Field | Description
+| `metadata.name` | The internal identifier for this secret. This name must match the value of the `secretRef.name` field of your Kubernetes Custom Resource.
+| `metadata.namespace` | Kubernetes namespace under which the secret is created, the supported value is `rtf`.
+| `data.persistence-gateway-creds` | Base64 encoded backend connection string.
+| `type` | Specifies the type of Kubernetes key. Only `Opaque` is supported.
+|===
+
+
+[[k8-custom-resource-template]]
+=== Kubernetes Custom Resource Template
+
+Use the example template below to create a Kubernetes custom resource. This custom resource enables the Kubernetes cluster to connect to your data store.
+
+----
+apiVersion: rtf.mulesoft.com/v1
+kind: PersistenceGateway
+metadata:
+  name: default
+  namespace: rtf
+spec:
+  objectStore:
+    backendDriver: postgresql
+    maxBackendConnectionPool: 20
+    replicas: 2
+    secretRef:
+      name: persistence-gateway-creds
+    resources:
+      limits:
+        cpu: 250m
+        memory: 250Mi
+      requests:
+        cpu: 200m
+        memory: 75Mi
+----
+
+Below are descriptions of the fields of the template you modify:
+
+[%header,cols="3*a"]
+|===
+| Field | Description | Default Value
+| `kind` | Specifies the kind of Custom Resource. Supported value is `PersistenceGateway`. | PersistenceGateway
+| `metadata.name` | The internal identifier for this Custom Resource. The value for this filed should be `default`. | default
+| `metadata.namespace` | The namespace where the secret is applied, the supported value is `rtf`. | rtf
+| `spec.objectStore.backendDriver` | Specifies the driver used by the data store. Only `postgresql` is supported. | postgresql
+| `spec.objectStore.maxBackendConnectionPool` | Specifies the maximum number of open connections to the backend at any given time. | 20
+| `spec.objectStore.replicas` | Specifies the number of replicas for Persistence Gateway deployment. | 2
+| `spec.objectStore.resources.limits.cpu` | Specifies the cpu resource limits for the Persistence Gateway pods. | 250m
+| `spec.objectStore.resources.limits.memory` | Specifies the memory resource limits for the Persistence Gateway pods. | 150Mi
+| `spec.objectStore.resources.requests.cpu` | Specifies the cpu resource requests for the Persistence Gateway pods. | 200m
+| `spec.objectStore.resources.requests.memory` | Specifies the memory resource requests for the Persistence Gateway pods. | 75Mi
+| `spec.objectStore.secretRef.name` | Specifies the name of the persistence gateway credentials defined in the Kubernetes secret file. | persistence-gateway-creds
+|===
+
+[NOTE]
+====
+The default CPU/Memory request and limit values are based on our internal testing with small number of deployed apps. Users can override these values as per their use case.
```

Comment:
@sundippatel @sarjeet2013 Please take a look at this `Note` section and suggest any corrections you have